src/javascripts/govuk-frontend.js

```diff
@@ -31,12 +31,6 @@ nodeListForEach($details, function ($detail) {
 var $errorSummaries = document.querySelectorAll('[data-module="govuk-error-summary"]')
 nodeListForEach($errorSummaries, function ($errorSummary) {
   var errorSummary = new ErrorSummary($errorSummary)
-  // Override the `init` method since it automatically focuses the ErrorSummary.
-  // This is not ideal when showing examples for this component
-  // TODO: Allow option for ErrorSummary to avoid this hack
-  errorSummary.init = function () {
```

Comment:
I _think_, now we're no longer needing to patch the init function, we should in theory be able to get rid of all of the individual imports and init calls and just use GOV.UK Frontend's `initAll` function? 🤔 

```
import { initAll } from 'govuk-frontend'
initAll()
```

However, if I try doing that locally, Rollup throws 'Error: 'initAll' is not exported by node_modules/govuk-frontend/govuk/all.js'.

The rollup docs [suggest this is a common issue when using the commonjs plugin](https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module). Again, testing locally, it looks like we can resolve this by adding it as a 'named export' in the commonjs config for govuk-frontend:

```js
    plugins: [
      // TODO: Since we're not exporting ES6 Modules to NPM we need to import these as commonjs
      resolve(),
      commonjs({
        namedExports: {
          'govuk-frontend': ['initAll']
        }
      })
    ]
```

I guess, based on the comment about ES6 from that same part of the code, the alternative is to merge as-is, and revisit once we've addressed https://github.com/alphagov/govuk-frontend/issues/711?

Reply:
@36degrees I've added it as a checklist item to https://github.com/alphagov/govuk-frontend/issues/711